### DISCUSSION TITLE
Will gphotos-sync also delete files localy when removed online?

### DISCUSSION THREAD

**Discussion by koen-cerm** [gilesknap/gphotos-sync#407] [Q&A] [answered]
Hi, 
I started using gphotos-sync today, and the sync worked like a charm. The reason why I started using it, is because my storage on google is getting full. My idea is to delete older photos on google photos, but use gphotos-sync first to download them locally. I wonder however if I run photos-sync again after I have removed my photos online on google photos: will gphotos-sync detect that the fotos are removed online, and thus also delete them locally? Or will it leave the local files. In my case it is the latter I would like. If not I must copy them first to another location to prevent deletion.

thanks,

Koen

**Comment by sebrem** [comment, marked as answer]
I think unless you explicitly use `--do-delete` the local photos are safe.

**Comment by koen-cerm** [comment]
Ah indeed, I didn't know about that flag. Thanks!
